src/core/Clock.js

```diff
@@ -14,9 +18,9 @@ function Clock( autoStart ) {
 
 }
 
-Object.assign( Clock.prototype, {
+Clock.prototype = {
 
-	start: function () {
+	start() {
```

Comment:
Removing the `: function` is required so that declaration emit works. The original post mentions a TODO item regarding this, for compat with old browsers.